src/components/BreakdownSection/CardMainNews.js

```diff
@@ -0,0 +1,29 @@
+import {
+  Card,
+  CardActionArea,
+  CardContent,
+  CardHeader,
+  CardMedia,
+  Typography,
+} from '@mui/material';
+import { styles } from './styles';
+
+export const CardMainNews = ({ category, image, title, sx = {} }) => {
+  return (
+    <Card sx={sx}>
+      <CardActionArea>
+        <CardHeader
+          title={category}
+          sx={styles.cardHeader}
+          titleTypographyProps={{ variant: 'subtitle1' }}
+        />
+        <CardMedia component='img' height={200} image={image} alt={title} />
+        <CardContent sx={{ height: '113px' }}>
```

Comment:
After adding flex props to the grid container in the breakdown section you should be able to remove height here.

Reply:
done :)